bot/exts/evergreen/issues.py

```diff
@@ -92,15 +109,72 @@ async def issue(
             issue_url = json_data.get("html_url")
             links.append([icon_url, f"[{repository}] #{number} {json_data.get('title')}", issue_url])
 
-        # Issue/PR format: emoji to show if open/closed/merged, number and the title as a singular link.
-        description_list = ["{0} [{1}]({2})".format(*link) for link in links]
+        return links
+
+    @staticmethod
+    def get_embed(result: list, user: str = "python-discord", repository: str = "") -> discord.Embed:
+        """Get Response Embed."""
+        description_list = ["{0} [{1}]({2})".format(*link) for link in result]
         resp = discord.Embed(
             colour=Colours.bright_green,
             description='\n'.join(description_list)
         )
 
         resp.set_author(name="GitHub", url=f"https://github.com/{user}/{repository}")
-        await ctx.send(embed=resp)
+        return resp
+
+    @commands.command(aliases=("pr",))
+    @override_in_channel(WHITELISTED_CHANNELS + (Channels.dev_contrib, Channels.dev_branding))
+    async def issue(
+            self,
+            ctx: commands.Context,
+            numbers: commands.Greedy[int],
+            repository: str = "sir-lancebot",
+            user: str = "python-discord"
+    ) -> None:
+        """Command to retrieve issue(s) from a GitHub repository."""
+        result = await self.fetch_issues(set(numbers), repository, user)
+
+        if result == "Numbers not found.":
```

Comment:
Using an enum for those results would be better, it would avoid having to change the string in many places if we want do want to change it.